Sources/VitalCore/Core/Client/Data Models/Patches/UserConnection.swift

```diff
@@ -131,7 +132,7 @@ public struct Provider: Equatable {
     case oura
     case garmin
     case fitbit
-    case whoop
+    case whoop_v2
```

Comment:
```suggestion
    case whoopV2
```

dexcom_v3 is missing as well